src/visualization/recipes_makie.jl

```diff
@@ -171,14 +175,14 @@ function Base.iterate(fa::FigureAndAxes, state=1)
 end
 
 function Makie.plot(pd::UnstructuredPlotData2D, fig=Makie.Figure();
-                    plot_mesh=true)
-  figAxes = Makie.plot!(fig, pd; plot_mesh=plot_mesh)
+                    plot_mesh=true, colormap=default_colormap())
+  figAxes = Makie.plot!(fig, pd; plot_mesh=plot_mesh, colormap=colormap)
   display(figAxes.fig)
   return figAxes
 end
 
 function Makie.plot!(fig, pd::UnstructuredPlotData2D;
-                     plot_mesh = true)
+                     plot_mesh = true, colormap)
```

Comment:
Not sure about the implications, but shouldn't this also get a default colormap?
```suggestion
                     plot_mesh=true, colormap=default_colormap())
```